specifications/api/swagger.yaml

```diff
@@ -7639,15 +7647,11 @@ paths:
 
   /blocks/latest/header:
     get:
+      operationId: getBlocksLatestHeader
       tags: ["Node"]
       summary: Block Header
       description: |
         <p align="right">status: <strong>stable</strong></p>
 
         Return the latest block-header available at the chain source
-      responses: 
-        200:
-          description: OK
-          content:
-            application/json:
-              schema: *ApiBlockHeader
+      responses: *responsesGetBlocksLatestHeader
```

Comment:
I actually advised @paolino not to extract the `x-responsesGetBlocksLatestHeader` as its only used in one place. 
My logic was that less indirection is better for the readability (local reasoning: you don't have to jump around in order to make sense of a piece of markup).

Reply:
ok, fixed.